.setup/bin/sample_courses/models/course/course_create_gradeables.py

```diff
@@ -0,0 +1,400 @@
+
+from __future__ import print_function, division
+from datetime import timedelta
+import hashlib
+import json
+import os
+import random
+import shutil
+import subprocess
+import os.path
+import random
+from tempfile import TemporaryDirectory
+from submitty_utils import dateutils
+
+from sqlalchemy import Table, select
+
+from sample_courses import *
+from sample_courses.utils import mimic_checkout
+from sample_courses.utils.dependent import commit_submission_to_repo
+from sample_courses.utils.create_or_generate import (
+    generate_probability_space,
+    generate_random_ta_comment,
+    generate_versions_to_submit,
+    generate_random_user_id,
+    create_gradeable_submission,
+    create_pdf_annotations
+    )
+from sample_courses.models.course.global_var import Table_var
+
+
+class Course_create_gradeables:
```

Comment:
I removed the inheritance from the class Course_create_gradeables because I'm already inheriting all the classes in the course class and if I inherit classes that are already inherited, I believe it loads double the classes. Also, I don't currently know what functions that are being called through the course class.